src/zeal/config.py

```diff
@@ -1,56 +1,59 @@
 import logging
 import os
 import platform
+from typing import Any
+import winreg
```

Comment:
`winreg` does not exist on non-windows systems. As such, this import needs to be in an if statement.

Reply:
yep, easy fix